install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -0,0 +1,188 @@
+[[configuring-etcd-monitoring]]
+= Configuring `etcd` monitoring
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+If the `etcd` service does not run correctly, successful operation of the whole OpenShift cluster is in danger. Therefore, it is reasonable to configure monitoring of `etcd`.
+
+Follow these steps to configure `etcd` monitoring:
+
+. Verify that the monitoring stack is running:
++
+[subs="quotes"]
+----
+$ *oc -n openshift-monitoring get pods*
+NAME                                           READY     STATUS              RESTARTS   AGE
+alertmanager-main-0                            3/3       Running             0          34m
+alertmanager-main-1                            3/3       Running             0          33m
+alertmanager-main-2                            3/3       Running             0          33m
+cluster-monitoring-operator-67b8797d79-sphxj   1/1       Running             0          36m
+grafana-c66997f-pxrf7                          2/2       Running             0          37s
+kube-state-metrics-7449d589bc-rt4mq            3/3       Running             0          33m
+node-exporter-5tt4f                            2/2       Running             0          33m
+node-exporter-b2mrp                            2/2       Running             0          33m
+node-exporter-fd52p                            2/2       Running             0          33m
+node-exporter-hfqgv                            2/2       Running             0          33m
+prometheus-k8s-0                               4/4       Running             1          35m
+prometheus-k8s-1                               0/4       ContainerCreating   0          21s
```

Comment:
STATUS for prometheus-k8s-1 is not running

Reply:
But the first replica (prometheus-k8s-0) is running, so the monitoring stack is running :)

Reply:
I mean as one example, we should make sure all the pods should be in running status